testsuite/MDAnalysisTests/data/coordinates/test.gro

```diff
@@ -0,0 +1,8 @@
+Written by MDAnalysis
+    5
+    1MET     CA    1   0.000   1.600   3.200  0.0000  0.1600  0.3200
```

Comment:
The last 3 values are the velocities. You should also check that they are activated in the tests